redhat-developer/developers.redhat.com
# [RHDX-567] Classification feed changes

### JIRA Issue Link
[RHDX-567](https://projects.engineering.redhat.com/browse/RHDX-567)

### Verification Process
The rhd_purpose_attributes module provides a route to view reports and a button for the csv download.  These changes introduces a permission to access the route and csv download.  The permission is added to the Content Admin role.

The view is updated for the csv export.  Any value that is empty now has "empty in Drupal". 

- Sign in as a `Content Creator`. Check the Configuration->User Interface menu and page. Verify Purpose Attributes menu link is not available. 
- Sign in as a `Content Admin`.  Verify the Purprose Attributes menu link is available. Verify you can download a CSV.  Check the CSV results and that they have 'empty in Drupal' and that there are no empty cells.
